core/common/src/test/java/alluxio/underfs/FingerprintTest.java

```diff
@@ -88,4 +89,31 @@ public void matchMetadataOrContent() {
     assertFalse(fp.matchContent(fpDataChanged));
     assertTrue(fp.matchMetadata(fpDataChanged));
   }
+
+  @Test
+  public void createFingerprintFromUfsStatus() {
+    String name = CommonUtils.randomAlphaNumString(10);
+    String owner = CommonUtils.randomAlphaNumString(10);
+    String group = CommonUtils.randomAlphaNumString(10);
+    short mode = (short) mRandom.nextInt();
+    String ufsName = CommonUtils.randomAlphaNumString(10);
+
+    UfsDirectoryStatus dirStatus = new UfsDirectoryStatus(name, owner, group, mode);
+    Fingerprint fp = Fingerprint.create(ufsName, dirStatus);
+    assertEquals(fp.getTag(Fingerprint.Tag.OWNER), owner);
```

Comment:
NIT: I think for `assertEquals`, the first argument should be the expected value, and the second argument is the one being tested. (same for the rest in this test method)

Reply:
Done!

Reply:
Can you change these batch of `assertEquals` too?